src/agent/communicator/src/communicator.cpp

```diff
@@ -41,12 +41,17 @@ namespace communicator
         {
             m_managerIp = getStringConfigValue("agent", "manager_ip");
             m_port = getStringConfigValue("agent", "agent_comms_api_port");
+            std::string strHttps =  getStringConfigValue("agent", "https_enabled");
```

Comment:
The string can be const. Also, since the type is already visible in this scope, there's no need for using Hungarian notation, it could be simply named "httpsEnabled" or "isHttpsEnabled".

Reply:
Done!